nearcore/src/state_sync.rs

```diff
@@ -125,7 +126,7 @@ impl StateSyncDumper {
                         dump_config.restart_dump_for_shards.clone().unwrap_or_default(),
                         external.clone(),
                         dump_config.iteration_delay.unwrap_or(Duration::seconds(10)),
-                        self.account_id.clone(),
+                        self.validator.get().map(|v| v.validator_id().clone()),
```

Comment:
I think this goes to a different thread so it should have the mutable config not a frozen one.